pulsar/consumer_multitopic.go

```diff
@@ -136,7 +136,11 @@ func (c *multiTopicConsumer) AckID(msgID MessageID) {
 }
 
 func (c *multiTopicConsumer) ReconsumeLater(msg Message, delay time.Duration) {
-	names, err := validateTopicNames(msg.Topic())
+	topic := msg.Topic()
```

Comment:
It looks like the issue is related to the individual partitioned topic name `my-topic-partition-0` is being configured in the consumer options instead of the partitioned name `my-topic`? If the partitioned topic name is passed in does this issue still exist?